floss/main.py

```diff
@@ -17,32 +18,19 @@
 import viv_utils
 import simplejson as json
 
-import version
-import strings
-import stackstrings
-import string_decoder
-import plugins.arithmetic_plugin
-import identification_manager as im
-import plugins.library_function_plugin
-import plugins.function_meta_data_plugin
-import plugins.mov_plugin
-from interfaces import DecodingRoutineIdentifier
-from decoding_manager import LocationType
-from base64 import b64encode
-
-from utils import get_vivisect_meta_info
-
+from . import __version__, logger as pkg_logger
+from . import strings, stackstrings, string_decoder, identification_manager as im
+from .const import MAX_FILE_SIZE, SUPPORTED_FILE_MAGIC, MIN_STRING_LENGTH_DEFAULT
+from .decoding_manager import LocationType
+from .interfaces import DecodingRoutineIdentifier
+from .plugins import arithmetic_plugin, library_function_plugin, function_meta_data_plugin, mov_plugin
+from .utils import get_vivisect_meta_info, hex
 
-floss_logger = logging.getLogger("floss")
 
-
-KILOBYTE = 1024
-MEGABYTE = 1024 * KILOBYTE
-MAX_FILE_SIZE = 16 * MEGABYTE
-
-SUPPORTED_FILE_MAGIC = set(["MZ"])
-
-MIN_STRING_LENGTH_DEFAULT = 4
+handler = logging.StreamHandler()
+handler.setLevel(logging.INFO)
+pkg_logger.addHandler(handler)
+logger = logging.getLogger(__name__)
```

Comment:
i think there is a `configure_logging` (or similarly named) function below, in which we do all the logging config. lets keep all these logic in the same place, so its easy to track. i'm aware that some users do rely on this function, so i'd prefer to avoid breaking the interface, if possible.